WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSTabEligibilityCheckerI2.swift

```diff
@@ -0,0 +1,260 @@
+import Combine
```

Comment:
I don't think we're using Combine here

Reply:
Good catch, removed in https://github.com/woocommerce/woocommerce-ios/pull/15865/commits/b6854d45b98ec369c0ada66e6c39d20e503129ed.